Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -8,18 +8,29 @@
 
 import UIKit
 
-class AddressWithTitleView: UINibView {
+struct AddressInfo {
+    let address: Address
+    let label: String?
+}
+
+class AddressInfoView: UINibView {
```

Comment:
I think this class is better to implement the copy and open in safari functionality by default; Plus, a bool flag to hide the "open in browser" button.

That way it will be truly reusable and we can just drop it in everywhere where needed

Reply:
I added copy by default. Let's discuss open in Safari by voice

Reply:
ok